week-1/Homework/extra/2-piping.js

```diff
@@ -16,26 +16,47 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(firstNum ,secondNum) {
+  const addition = firstNum +secondNum;// adds to number
+  const roundify = Math.round(addition*100)/100;// rounds the result
```

Comment:
Do we need to do `(addition*100)/100` here? Will it work without it? If we need it, why is that so?